hdinsight-linux-with-hue-application/README.md

```diff
@@ -0,0 +1,26 @@
+# Create HDInsight Linux Cluster with Edge Node and Hue installed on that Edge Node
+
+Create HDInsight Linux Cluster with Edge Node and Hue app installed on Edge Node -<br>
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fhdinsight-linux-with-hue-application%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fhdinsight-linux-with-hue-application%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+Template installs Hue as an Iaas Cluster Application on the edge node of an existing cluster. 
```

Comment:
This readme looks almost the same as that for hdinsight-linux-add-hue-on-existing-cluster template?